Models/DTOs/RatingInputDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace AonFreelancing.Models.DTOs
+{
+    public class RatingInputDTO
+    {
+        public long RatedUserId { get; set; }
+        [Range(1, 10)]
+        public double RatingValue { get; set; }
```

Comment:
PLEASE, rename this property to a more descriptive name such as Score.

Reply:
Noooooooooooo .... leave it alone :(